python/singa/autograd.py

```diff
@@ -1828,3 +1828,18 @@ def backward(self, dy):
 
 def leakyrelu(x, a=0.01):
     return LeakyRelu(a)(x)[0]
+
+
+class Sign(Operation):
+    def forward(self, a):
```

Comment:
Add the default initializer:

    def __init__(self):
        super(Sign, self).__init__()

Reply:
fix the code

Reply:
Thanks for the modification. The below is the reason autograd.py this is needed.

This function passes the name "Sign" to the superclass (Operation) initialize.
The superclass Operation initialize is as follows:
```
class Operation(object):
    op_count = 0
    def __init__(self, name=None):
        if name is None:
            self.name = "{}#{}".format(
                self.__class__.__name__, Operation.op_count
            )
            Operation.op_count += 1
        else:
            self.name = name
```
If there is no initialzer in the subclass, the superclass initializer will be called with the "name=None"